src/static/css/almanac.css

```diff
@@ -389,6 +389,35 @@ h2.header {
   text-align: center;
 }
 
+.btn.chapter-cta > svg {
+  margin: 0px 5px;
+}
+
+.chapter-cta {
+  position: fixed;
+  border: none;
+  box-shadow: 2px 2px 3px 3px lightgray;
```

Comment:
Should we apply the `.floating-card` class and reuse its box-shadow styles for consistency?